scripts.js

```diff
@@ -0,0 +1,38 @@
+(function () {
+  console.log('hey');
```

Comment:
I'm wondering what this line of code does. I'm thinking it was just to make sure your code was working. Do we still need it?